src/DynamoUtilities/Hash.cs

```diff
@@ -0,0 +1,56 @@
+using System.Security.Cryptography;
+using System.Text;
+
+namespace Dynamo.Utilities
+{
+    internal class Hash
+    {
+        /// <summary>
+        /// Get the hash value
+        /// </summary>
+        /// <param name="bytes">input as a byte array</param>
+        /// <returns>hash as a byte array</returns>
+        internal static byte[] GetHash(byte[] bytes)
+        {
+            using (var hashAlgorithm = HashAlgorithm.Create())
```

Comment:
what algorithm is this? Any idea how likely collisions are?

Reply:
Good point. I did not think to much about that. The default implementation is SHA1. SHA1 have known collision problems. I also noticed that this create method is deprecated in Net 5 and later. I will change this to SHA256 instead. Thanks!